### PR TITLE
V8.x - Update library to use serato/jwt 3.0

### PR DESCRIPTION
**Pull request by Mike-Serato**
- Was unsure why a Memcached instance was required to bootstrap the library when a PSR Cache Pool should be enough? Let me know if I am missing anything.
- Updated unit tests for Access token middleware.
- Also includes some cleanup like adding return types, updating Docblocks etc.

### CODE REVIEW

File: src/Slim/Middleware/AccessScopes/AccessToken.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ class AccessToken extends AbstractAccessScopesMiddleware
      *
      * @var Sdk
```
Comment (jordan-chen-serato) [inline src/Slim/Middleware/AccessScopes/AccessToken.php:30]: Should change the comment.

File: src/Slim/Middleware/AccessScopes/AccessToken.php [hunk 2]
```diff
@@ -186,11 +173,11 @@ private function getAccessTokenScopes(JwtAccessToken $accessToken): array
     /**
      * Get the AWS Sdk
```
Comment (jordan-chen-serato) [inline src/Slim/Middleware/AccessScopes/AccessToken.php:174]: Change the comment

File: src/Slim/Middleware/AccessScopes/AccessToken.php [hunk 3]
```diff
@@ -2,11 +2,11 @@
 
 namespace Serato\SwsApp\Slim\Middleware\AccessScopes;
 
+use Aws\Kms\KmsClient;
 use Aws\Sdk;
```
Comment (jordan-chen-serato) [inline src/Slim/Middleware/AccessScopes/AccessToken.php:6]: Do we still need this?

File: src/Slim/Middleware/AccessScopes/ClientAppBasicAuth.php [hunk 4]
```diff
@@ -52,6 +51,7 @@ public function __construct(string $webServiceName, array $clientAppData)
      * @param Request $request The most recent Request object
      * @param Response $response The most recent Response object
      * @param callable $next
+     * @return mixed
```
Comment (jordan-chen-serato) [inline src/Slim/Middleware/AccessScopes/ClientAppBasicAuth.php:54]: Is this intended?

**Review by jordan-chen-serato — CHANGES_REQUESTED** [review]

File: src/Slim/Middleware/AccessScopes/ClientAppBasicAuth.php (hunk 4 above)
Comment (Mike-Serato) [reply src/Slim/Middleware/AccessScopes/ClientAppBasicAuth.php:54]: Yes, it is. $next is a callable.

**Review by harrietrc — APPROVED** [review]
Looks great! I agree that the Memcached instance isn't necessary. It looks like it was redundant before, and Chris has switched to using `CacheItemPoolInterface::getIem()` in place of `Memcached::get()` everywhere in the new release.

**Review by jordan-chen-serato — APPROVED** [review]

**Pull request merged**